api/views.py

```diff
@@ -400,7 +401,14 @@ def get(cls, request):
 class ProjectPrimarySectors(APIView):
     @classmethod
     def get(cls, request):
-        keys_labels = [{'key': i, 'label': v} for i, v in Sectors.choices]
+        keys_labels = [{'key': s.id, 'label': s.title} for s in Sector.objects.all()]
+        return JsonResponse(keys_labels, safe=False)
+
+
+class ProjectSecondarySectors(APIView):
+    @classmethod
+    def get(cls, request):
+        keys_labels = [{'key': s.id, 'label': s.title} for s in SectorTag.objects.all()]
```

Comment:
There are some really similar queries, it's true, but I could not make it under "one hood".
On the other hand maybe some extra data will be needed to be shown here (e.g. color), to help the frontend development: https://github.com/IFRCGo/go-frontend/issues/2665
Anyway, if you suggest a nicer way, I am really open to it!

Reply:
I would suggest one API to send all static + dynamic(like static). To avoid requests latency between requests and [wait time](https://kb.mozillazine.org/Network.http.max-persistent-connections-per-server).
But I am not aware of what decision we made before 😄 .

NOTE: We could add a filter using GET parameters and also do server-side memory cache if required.